tox.ini

```diff
@@ -59,3 +59,10 @@ commands =
   sphinx-build -j auto -W -T --keep-going {posargs} docs/ docs/_build/html
 passenv =
   CI
+
+[testenv:docs-clean]
+skip_install = true
+allowlist_externals =
+  rm
+commands =
+  rm -rf {toxinidir}/docs/stubs/ {toxinidir}/docs/_build/
```

Comment:
Ack, I noticed that. It only deletes `.doctrees` inside `html` rather than the entire folder, however. I also think it's nice to have `tox -e docs-clean` to be consistent with the other addon repos - muscle memory.

Lmk if you want me to delete this though. I'm fine with that

Reply:
Sorry, I meant that we could potentially remove those steps from the `docs` environment now that there is docs-clean.